tlsn/examples/twitter/twitter_dm.rs

```diff
@@ -34,21 +32,8 @@ async fn main() {
     let access_token = env::var("ACCESS_TOKEN").unwrap();
     let csrf_token = env::var("CSRF_TOKEN").unwrap();
 
-    let (notary_tls_socket, session_id) =
-        request_notarization(NOTARY_HOST, NOTARY_PORT, None, None).await;
-
-    // Basic default prover config using the session_id returned from /session endpoint just now
-    let config = ProverConfig::builder()
-        .id(session_id)
-        .server_dns(SERVER_DOMAIN)
-        .build()
-        .unwrap();
-
-    // Create a new prover and set up the MPC backend.
-    let prover = Prover::new(config)
-        .setup(notary_tls_socket.compat())
-        .await
-        .unwrap();
+    // Create a new prover
+    let prover = request_notarization(NOTARY_HOST, NOTARY_PORT, None, None, SERVER_DOMAIN).await;
```

Comment:
The demonstration is in examples/src/lib.rs — mainly to avoid code duplication in different examples, or you see some issues/downsides with this approach?